src/taipy/core/cycle/_cycle_manager.py

```diff
@@ -11,21 +11,24 @@
 
 import calendar
 from datetime import datetime, time, timedelta
-from typing import List, Optional
+from typing import Callable, Dict, List, Optional, Union
 
 from taipy.config.common.frequency import Frequency
 
 from .._manager._manager import _Manager
+from .._repository._v2._abstract_repository import _AbstractRepository
+
+# from .._repository._v2._filesystem_repository import _FileSystemRepository
+from .._version._version_mixin import _VersionMixin
 from ..common._entity_ids import _EntityIds
 from ..common.alias import CycleId
 from ..job._job_manager_factory import _JobManagerFactory
-from ._cycle_repository_factory import _CycleRepositoryFactory
 from .cycle import Cycle
 
 
-class _CycleManager(_Manager[Cycle]):
-    _repository = _CycleRepositoryFactory._build_repository()  # type: ignore
+class _CycleManager(_Manager[Cycle], _VersionMixin):
```

Comment:
Cycle entity doesn't include version, so is this necessary?

Reply:
You're right. I have remove it